crates/sui-core/src/authority_server.rs

```diff
@@ -288,7 +288,7 @@ impl ValidatorService {
                 .start_timer()
         };
 
-        let epoch_store = state.epoch_store();
+        let epoch_store = state.load_epoch_store_one_call_per_task();
```

Comment:
this is a bit spooky - do you know if there's a reason we don't acquire the reconfig lock at the very top of this function instead of further down?

Reply:
Moved it up a bit. Regarding your question, we only need to hold the reconfig lock when we interact with consensus.